app/views/mailer/mfa_notification.html.erb

```diff
@@ -0,0 +1,64 @@
+<% @title = "Please enable MFA" %>
+<% @sub_title = "Hi #{@user.handle}" %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="40" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px; text-align:center">
+        We care about keeping your RubyGem.org account secure and recommend enabling multifactor authentication for <b>both UI and API</b>.
+        Setting up MFA only takes a few minutes and goes a long way towards protecting your account from any malicious actor.
+        <br/>
```

Comment:
Mentioning this helps protect the Ruby ecosystem might persuade a few more users to enable MFA, e.g.

> Setting up MFA only takes a few minutes and goes a long way towards protecting your account and the Ruby ecosystem.